bindings/go/mio/mio.go

```diff
@@ -341,25 +341,31 @@ const maxM0BufSz = 128 * 1024 * 1024
 // Estimate the optimal (block, group) sizes for the I/O
 func (mio *Mio) getOptimalBlockSz(bufSz int) (bsz, gsz int) {
     if bufSz > maxM0BufSz {
-            bufSz = maxM0BufSz
+        bufSz = maxM0BufSz
     }
     pver := C.m0_pool_version_find(&C.instance.m0c_pools_common,
                                    &mio.obj.ob_attr.oa_pver)
     if pver == nil {
-            log.Panic("cannot find the object's pool version")
+        log.Panic("cannot find the object's pool version")
     }
-    usz := int(C.m0_obj_layout_id_to_unit_size(C.ulong(mio.objLid)))
     pa := &pver.pv_attr
-    gsz = usz * int(pa.pa_N)
-    /* max 2-times pool-width deep, otherwise we may get -E2BIG */
+    if pa.pa_P < pa.pa_N + 2 * pa.pa_K {
+        log.Panic("pool width (%v) is less than the parity group size" +
+                  " (%v + 2 * %v == %v), check pool parity configuration",
+                  pa.pa_P, pa.pa_N, pa.pa_K, pa.pa_N + 2 * pa.pa_K)
+    }
```

Comment:
In LR2, we are going to handle N+k+0 configuration (No spare units).

What should the go binding do after that feature is done?

Reply:
I suppose in this case we will have pa_S field in the m0_pdclust_attr structure or something? If so - we could use it here then.

Any other idea / suggestion?

Reply:
This task is planned. Detailed design/implementation is still in progress.

So, in future, when that is ready, this file needs to be updated.

Reply:
Yes.